tcx/src/lib.rs

```diff
@@ -297,40 +297,39 @@ mod tests {
+    //                    let err = _to_str(get_last_err_message());
     //
-    //                println!("{:?}", Response::decode(err_bytes).unwrap());
-    //                assert!(false);
-    //            });
-    //        }
+    //                    let err_bytes = hex::decode(err).unwrap();
+    //
+    //                    println!("{:?}", Response::decode(err_bytes).unwrap());
+    //                    assert!(false);
+    //                });
+    //            }
 
-    //    #[test]
-    //    fn test_encode_empty_struct() {
-    //        //        let param: KeystoreCommonExistsResult = KeystoreCommonExistsResult {
-    //        //            is_exists: false,
-    //        //            id: "".to_string()
-    //        //        };
-    //        //        let hex_value = hex::encode(encode_message(param).unwrap());
-    //        //        assert_eq!("08001200", hex_value);
-    //        let bytes = hex::decode("1211756e737570706f727465645f636861696e").unwrap();
-    //        let rsp = Response::decode(bytes);
-    //        println!("{:?}", rsp);
-    //        //        let param: KeystoreCommonExistsResult = KeystoreCommonExistsResult::decode(bytes).unwrap();
-    //        //        let param2: KeystoreCommonExistsResult =
-    //        //            KeystoreCommonExistsResult::decode(vec![]).unwrap();
-    //        //        assert_eq!(param.is_exists, param2.is_exists);
-    //    }
+    //        #[test]
+    //        fn test_encode_empty_struct() {
+    //            //        let param: KeystoreCommonExistsResult = KeystoreCommonExistsResult {
+    //            //            is_exists: false,
+    //            //            id: "".to_string()
+    //            //        };
+    //            //        let hex_value = hex::encode(encode_message(param).unwrap());
+    //            //        assert_eq!("08001200", hex_value);
+    //            let bytes = hex::decode("0a1174726f6e5f7369676e5f6d65737361676512d5010a1174726f6e5f7369676e5f6d65737361676512bf010a2439346434376666642d366631382d343665302d613962652d373132646535363066363132120e31323331323331323321402324251a0454524f4e222254593275726f42655a3574724139515439366145576a3332584c6b414168513952322a5d0a1174726f6e5f7369676e5f6d65737361676512480a4230783634356330623762353831353862616262666136633663643561343861613733343061383734393137366231323065383531363231363738376131336463373610011801").unwrap();
+    //            let param = KeystoreCommonExistsParam::decode(bytes).unwrap();
+    ////            let rsp = Response::decode(bytes);
+    //            println!("{:?}", param);
+    //            //        let param: KeystoreCommonExistsResult = KeystoreCommonExistsResult::decode(bytes).unwrap();
+    //            //        let param2: KeystoreCommonExistsResult =
+    //            //            KeystoreCommonExistsResult::decode(vec![]).unwrap();
+    //            //        assert_eq!(param.is_exists, param2.is_exists);
+    //        }
```

Comment:
plz remove the unused code

Reply:
It's used for  debug protobuf quickly and it's quite useful when debug. I can remove it if it bores someone.